site/src/api/api.ts

```diff
@@ -282,7 +282,16 @@ export const suspendUser = async (userId: TypesGen.User["id"]): Promise<TypesGen
   return response.data
 }
 
-export const postFirstUser = async (
+export const hasFirstUser = async (): Promise<boolean> => {
+  try {
+    await axios.get("/api/v2/users/first")
+    return true
```

Comment:
Interesting that this response doesn't just return true or false. Given this is a bit of an unconventional route can we add a comment? Also, can we use the typegen `CreateFirstUserResponse`?

Reply:
1. Sure
2. Nops, because the response for this one does not return a useful object. Basically, 404 is false and 200 is true.